python/GafferSceneUI/StandardOptionsUI.py

```diff
@@ -96,281 +97,422 @@ def __statisticsSummary( plug ) :
 
 	return ", ".join( info )
 
-Gaffer.Metadata.registerNode(
+renderOptionsMetadata = {
```

Comment:
Don't know if I'm missing somthing, but do we need to split the metadata up like this (between `renderOptionsMetadata` and `plugMetadata`)? Is it just because they have the `.options` prefix on this node? Seems fragile and confuses what the source of truth is and why it's separate? Took me 10 mins to work out what the separation was reading from cold. Could just put it all in one?

The code in other classes still works, it just becomes `renderOptionsMetadata["options.filmFit"]` instead of `["filmFit"]`.  The transformation in `CameraUI` becomes:
`overrideKeyName = keyName.replace( "options.", "renderSettingOverrides." )` + a little logic to find the `descriptions` entry, afaict.

Reply:
Digesting some more - if you're concerned about having to filter out layout sections etc... then maybe at least make this just a simple descriptions dictionary and put _all_ the descriptions in. So its `{ <name> : <description }`, and have `renderOptionsPresets` for `filmFit` which seems to be the only one that shares anything other than a description.  It's annoying we're suck with the old python bindings that mean you have paired values in an array.

We just don't want to have `StandardOptions` metadata being split up in to parts just because `CameraUI` wants to access some of the data. That might change in the future and StandardOptions shouldn't have to be updated if say, we decide to expose `motionBlur` as an override.... so if we're making the concession for sharing, then at least make sure we're consistent within this file about how we handle metadata... That split of why some plugs are in there and some aren't take a bit of reverse engineering to understand.

Reply:
Fair points. Partly I was just following what you suggested, and partly I'm just not fully aware of all the methods available for manipulating Python dicts. I'll improve it.

Reply:
Addressed in 1c2a4237a4d47008ec2014616029106b956c7b1c.